identity-service/src/routes/trackListens.js

```diff
@@ -407,7 +408,7 @@ module.exports = function (app) {
     const { trackIdList } = req.query
 
     if (!trackIdList || !Array.isArray(trackIdList)) {
```

Comment:
I think you can remove the first part b/c `Array.isArray(undefined)` should be false.
don't need to change though

Reply:
while that's true, I think this is more clear this way and it's faster in the more common case where it's just left out